### ISSUE TITLE
Unclear information for how `tolerateNonExistentBackendService` and `tolerateNonExistentBackendAction` mechanism work

### ISSUE THREAD

**Issue by guessi** [kubernetes-sigs/aws-load-balancer-controller#3440]
**Is your feature request related to a problem?**

It is hard to understand how `tolerateNonExistentBackendService` and `tolerateNonExistentBackendAction` mechanism works without tracing code changes.

As an user, a typo of "use-annotations" (suppose to be "use-annotation") would lead to a 503 error but not knowing why? until tracing the source code then I can see it's all about the following code block behavior.

```
func (b *defaultEnhancedBackendBuilder) buildActionViaAnnotation(...) (...) {
    ...
	if !exists {
		if b.tolerateNonExistentBackendAction {
			return b.build503ResponseAction(nonExistentBackendActionMessageBody), nil
		}
		return Action{}, errors.Errorf("missing %v configuration", annotationKey)
	}
    ...
}
```

* [aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L20-L31](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L20-L31)
* [aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L96-L108](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L96-L108)
* [aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L175-L195](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.6.1/pkg/ingress/enhanced_backend_builder.go#L175-L195)

Moreover, it is configurable via Helm chart (main branch)
* https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/helm/aws-load-balancer-controller/values.yaml#L230-L234

But not available in v2.6.1 version Helm chart,
* https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.6.1/helm/aws-load-balancer-controller/values.yaml

Checked current latest [eks-charts](https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller), not available as well,
* https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/values.yaml

**Describe the solution you'd like**

Documentation improvement for how `tolerateNonExistentBackendService` and `tolerateNonExistentBackendAction` mechanism work.

**Describe alternatives you've considered**

n/a

**Comment by oliviassss** [comment]
@guessi, thanks for the details. It's not available in v2.6.1 helm chart is because we haven't released it yet. We plan to release this feature in a coming patch release.

**Comment by guessi** [comment]
Close, as #3442 have been merged.
